src/file/file.service.ts

```diff
@@ -178,8 +175,7 @@ export class FileService {
     let files: IFile[];
     if (!folderID) { // Search the user's root folder
       if (!ownerID) throw new ClientError('No file or owner id sent');
-      const rootFolder = await this.findUserRootFolder(ownerID);
-      files = await FilesRepository.find({ deleted, parent: rootFolder });
+      files = await FilesRepository.find({ deleted, parent: null });
     } else files = await FilesRepository.find({ deleted, parent: folderID });
```

Comment:
This doesn't work since `folderID` is a string and `parent` is of type `ObjectId`, so if we want to add folders support then querying files within folders wouldn't work.